packages/common/components/gam/prestitial.marko

```diff
@@ -0,0 +1,40 @@
+import { getAsObject } from "@base-cms/object-path";
+
+$ const { site } = out.global;
+$ const { adunit, slotId } = input;
+
+<if(adunit)>
+  <if(input.init)>
+    $ const targeting = { ...getAsObject(adunit, 'targeting'), ...getAsObject(input, 'targeting') };
+    $ const slots = {
+      [slotId]: {
+        path: adunit.path,
+        size: adunit.size,
+        sizeMapping: adunit.sizeMapping,
+        targeting,
+      },
+    };
+    <marko-web-gam-slots slots=slots />
+    <prestitial-script />
+  </if>
+  <if(input.display)>
+    $ const logoSrc = site.get("logos.prestitial.src", site.get("logos.footer.src"));
+    <div
+      data-interstitial-prop="bg-url"
+      data-interstitial-value="https://base.imgix.net/files/base/ebm/global/static/welcome_1200_alt2.png"
+      style="display: none;"
+    />
+    <!-- The id is being kept to prevent the create template from erroring. -->
+    <!-- @todo Update the creative template to not use this. -->
+    <div
+      id="wl_flogop_div"
+      data-interstitial-prop="logo-url"
+      data-interstitial-value=logoSrc
+      style="display: none;"
+    />
+    <marko-web-gam-out-of-page-ad ...adunit id=slotId style="height: 0;" />
```

Comment:
Was the zero height also set in the informa repo?

Reply:
yep